ethwrite/README.md

```diff
@@ -0,0 +1,70 @@
+# Chainlink External Adapter for writing to Ethereum-based Blockchains
+
+This external adapter allows you to configure an endpoint and private key to sign and send transactions to external Ethereum-based blockchains.
+
+A typical workflow of a Chainlink job for this external adapter could look like:
+
+- Retrieve a piece of data from _some data source_
+- Parse the desired field from that data source's response
+- Utilize this adapter to write the value to ChainB
+- Parse the transaction object from ChainB for the transaction hash
+- Write the transaction hash from ChainB to ChainA
+
+## Input Params
+
+- `exAddr`: The address for sending the transaction to.
+- `funcId`: (Optional) One of the following setter functions: `0xc2b12a73` for `bytes32`, `0xa53b1c1e` for `int256` or `0xd2282dc5` for `uint256` (defaults to `0xd2282dc5`)
+- `dataType`: (Optional) One of the following types: `bytes32`, `int256` or `uint256` (defaults to `uint256`)
```

Comment:
We normally get the data pre-encoded from the previous tasks in the job run, and IMO we should not have dataType as a param (it should be assumed it's already encoded)

Reply:
Made it optional(default usage will not encode the result), to retain the other functionality as well.